src/maps/event/mvcobject.ts

```diff
@@ -17,6 +17,12 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
 
 export class MVCObject implements google.maps.MVCObject {
+  public static mockInstances: MVCObject[] = [];
+
+  public constructor() {
+    MVCObject.mockInstances.push(this);
```

Comment:
would this result in the following?

```js
new MVCObject();
new Map();

Map.mockInstances.length === 2
```

Reply:
I think this https://stackoverflow.com/a/20851359/3145360 might provide a solution?

Reply:
https://github.com/googlemaps/js-jest-mocks/pull/176/commits/30c89d418e31cc9baf0dc3429b2ed2e276db89a9

Reply:
Still verifying though

Reply:
Oh, I see what you mean.  Okay yeah, fixing :)

Reply:
Hm, by doing it this way, I'm not sure how to implement the auto clearing functionality of `afterEach`.  There is no one class to reset the static vars on anymore.

Reply:
Can manually maintain the list of child classes or use some other mechanism for tracking the arrays?